tests/app/core/models/session-model.spec.ts

```diff
@@ -83,7 +71,7 @@ describe('Session Model', () => {
     });
 
     it('should not throw error if session data contains repo information of unopened phases', () => {
-      const modifiedSessionData: SessionData = { ...validSessionData, openPhases: [Phase.phaseBugReporting] };
+      const modifiedSessionData: SessionData = { ...MODERATION_PHASE_SESSION_DATA, openPhases: [Phase.phaseBugReporting] };
```

Comment:
Wouldn't this be the same as just using `MODERATION_PHASE_SESSION_DATA`?

Reply:
Eyes, I just replaced the variable cause I didn't take too much notice to the test 😅  mbmb!

Reply:
Haha no worries man 👍